packages/connectivity/src/scp-cf/destination/service-binding-to-destination.ts

```diff
@@ -56,6 +56,24 @@ export async function transformServiceBindingToDestination(
   );
 }
 
+/**
+ * Transforms a service binding to a destination of type OAuth2ClientCredentials.
+ * @param serviceBinding - The service binding to transform.
+ * @param options - Options used for fetching the destination.
+ * @returns A promise returning the transformed destination on success.
+ */
+export async function transformServiceBindingToClientCredentialsDestination(
+  serviceBinding: Service,
```

Comment:
```suggestion
  service: Service,
```

[pp] To keep it consistent with other functions